client/src/components/SocialMediaIndexPage.js

```diff
@@ -1,13 +1,12 @@
 import React from "react";
-import { BrowserRouter } from "react-router-dom";
 import SocialMediaList from "./SocialMediaList";
 
-const SocialMediaIndexPage = (props) => {
-    return (
-        <div className="grid-container">
-                <SocialMediaList />
-        </div>
-    )
+const SocialMediaIndexPage = () => {
+  return (
+    <div className="grid-container">
+      <SocialMediaList />
+    </div>
+  );
 };
```

Comment:
this component is largely pointless. Delete it and just render `SocialMediaList` at this path